### ISSUE TITLE
[Bug]: Customized Project Text value doesn't update when switching files.

### ISSUE THREAD

**Issue by markshust** [mallowigi/material-theme-issues#293]
### Describe the bug

Re: this new feature: https://github.com/mallowigi/material-theme-issues/issues/287

When toggling between files, the value doesn't update. See video:

https://user-images.githubusercontent.com/437029/210375207-43799112-7c3a-436b-808a-83d2c75190f8.mov

### To Reproduce

1. Enable customized path and set to `{path}`
2. Switch between more than 1 file.

### Expected behavior

Value of `{path}` updates when switching files to reflect the current file

### Material Theme Version

8.4.1

### IDE

PHPStorm

### System Info

```shell
Build #PS-223.8214.64, built on December 22, 2022

Licensed to Mark Shust
Subscription is active until May 8, 2023.

Runtime version: 17.0.5+1-b653.23 aarch64
VM: OpenJDK 64-Bit Server VM by JetBrains s.r.o.
```

**Comment by markshust** [comment]
Hi @mallowigi,

I just wondered what fix version this will be in and when it'll be pushed out? I'm on 8.4.1 and this is still happening for me.

Cheers,
Mark

**Comment by mallowigi** [comment]
That's right, it's on 8.4.2 which is only available in 2023.1 

I don't want to continue working on 2022.3 as this will make versioning a hassle, so if you can keep working with this it would be great.

Or download the eap :)

**Comment by markshust** [comment]
Ok no prob at all, I'll test out the EAP. Thanks!
